packages/atomic/package.json

```diff
@@ -66,7 +85,7 @@
   "devDependencies": {
     "@axe-core/playwright": "4.9.1",
     "@babel/core": "7.24.9",
-    "@coveo/atomic": "file:.",
```

Comment:
We don't want to have the storybook utils in our "official" exports, so let's tweak this like so